_docs/3.0.0/usage.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+title: Usage
+version: '3.0.0'
+categories:
+    - '3.0.0'
+---
+{% capture docs_path %}{{ site.baseurl }}/docs/{{ page.version }}{% endcapture %}
+
+## Table Of Contents
+
+<!-- toc -->
+- [Table Of Contents](#table-of-contents)
+- [Intro](#intro)
+- [Quick Start](#quick-start)
+- [Spark Jobs](#spark-jobs)
+- [Menas](#menas)
+- [General](#general)
+- [Details](#details)
+<!-- tocstop -->
+
+## Intro
+
+This part of the documentations will show you how to use Enceladus, Menas as its UI and how to run its spark-jobs. This part expects you to have Menas already deployed and running and `spark-jobs.jar` ready at hand. If not, please look at previous parts of [build process][build] and [deployment][deploy]
+
+## Quick Start
+
+- [Menas Quick Start]({{ docs_path }}/usage/menas-quick-start)
+- [Data & Data Quality Quick Start]({{ docs_path }}/usage/data-quick-start)
+- [Spark Jobs Quick Start]({{ docs_path }}/usage/spark-jobs-quick-start)
+
+## Spark Jobs
+
+- [Configuration]({{ docs_path }}/usage/config)
+- [Standardization Input Formats]({{ docs_path }}/usage/standardization-formats)
+
+## Menas
+
+- [Configuration]({{ docs_path }}/usage/menas-config)
+- [Conformance Rules]({{ docs_path }}/usage/menas-conformance-rules)
+- [API]({{ docs_path }}/usage/menas-api)
```

Comment:
```suggestion
```

As outlined elsewhere, let's remove this and later, in #2092 of GHP, let's include a page about Enceladus REST API (v3) referencing to the built-in Swagger doc.